app/controllers/change_requests_controller.rb

```diff
@@ -27,11 +27,11 @@ def index
     end
     @has_archived = false
 
-    @change_requests_requested = @user.change_requests
+    @change_requests_requested = ChangeRequests.all_change_requests(@user).where('requestor_id = ?', @user.id)
```

Comment:
This is supposed to be ChangeRequest was a typo on my part

Reply:
@mcrutch Fixed